src/tab-manager/__tests__/index.test.js

```diff
@@ -38,7 +40,14 @@ describe('Tab Manager module test suite', () => {
       MenuItem: jest.fn(() => mockMenuItem),
       session: {fromPartition: jest.fn(() => ({}))}
     }));
-    jest.mock('../../settings');
+    mockSettings = {
+      loadSettings: jest.fn(() => ({
+        tabs: [{id: '1337', notificationsEnabled: true}],
+        notifications: true
+      })),
+      updateSettings: jest.fn()
+    };
```

Comment:
Logic for notifications must be "reversed". Right now notifications are in an opt-in fashion, but default should be opt-out (I want to be notified by default, I will provide a setting in case I don't want to).

This will also provide backwards configuration for previous setting files. With the current status, any user upgrading to the newest version of ElectronIM will loose notifications without knowing it.

Properties should be renamed (and logic changed accordingly) to:
```suggestion
    mockSettings = {
      loadSettings: jest.fn(() => ({
        tabs: [{id: '1337', disableNotifications: true}],
        disableNotificationsGlobally: true
      })),
      updateSettings: jest.fn()
    };
```

Reply:
Done the entire refactor for this change.
I also included one more test to test this property of the setting (opt-out), assuring that the notifications will show even if the settings still don't have these 2 new parameters.